README.md

```diff
@@ -75,6 +76,10 @@ This tool is built on top of the [oclif](https://oclif.io/) library to provide C
 
 It interfaces with the GitHub API to carry out the necessary [steps](###steps) for the migration (more information below). It makes use of the [octokit](https://github.com/octokit/rest.js/) library.
 
+### Dry Run Mode
+
+The dry run mode can be used to see what would happen if the changes were made. The process will perform an GET requests and log all steps as if they were to be executed. This can be useful both to see how the process works without making changes and to run checks to reduce the change of encountering and error partway through the process.
```

Comment:
```suggestion
The dry run mode can be used to see what would happen if the changes were made. The process will perform GET requests and log all steps as if they were to be executed. This can be useful both to see how the process works without making changes and to run checks to reduce the chance of encountering and error partway through the process.
```